### ISSUE TITLE
Git not working with Private Gitlab on Custom port

### ISSUE THREAD

**Issue by ryo-osive** [litespeedtech/cyberpanel#342]
I am not able to make git work with my private gitlab since its on custom port i.e. example.com:9008.
Since the text field accepts only domain name and **:** is not accepted value.

![image](https://user-images.githubusercontent.com/63479117/82265285-445e8780-9984-11ea-8b04-e68728d9f9ed.png)

**Comment by usmannasir** [comment]
Interesting find, will fix soon.

**Comment by usmannasir** [comment]
Fixed.

Get the fix using from a current beta branch (v2.0.1)

```
wget https://raw.githubusercontent.com/usmannasir/cyberpanel/v2.0.1/cyberpanel_upgrade.sh
chmod +x cyberpanel_upgrade.sh
./cyberpanel_upgrade.sh
```

Enter version 2.0.1 when asked.

**Comment by ryo-osive** [comment]
I am running my cyberpanel on Google Cloud and want to fix this issue without a chance of breaking anything.
Is it possible that I can Upgrade?

**Comment by usmannasir** [comment]
Safe to upgrading using commands above. If you run into any issues contact our support team at https://platform.cyberpanel.net/

**Comment by bagus-rataha** [comment]
Hello, i know its been a while since the issue is closed and completed. But i've facing the similar problem to this issue on my self hosted cyberpanel. So im able to put the host and port on the input field `subdomain.domain.com:customport` like the picture, but when im trying to attach, the reponse show that it still hit the default port 22. 

![Image](https://github.com/user-attachments/assets/2b36150a-dfd6-4068-a340-66f701289f73)


Ive tried using the safe upgrade command as shown above, but I think it is no longer relevant because the OS version supported in that command is only for a few OS versions, while I am using Ubuntu 22.04.5

![Image](https://github.com/user-attachments/assets/b3ab4bed-d236-4834-b3b8-42f866184531)



How can i solve this issue ? is there anything that im missing ? fyi im using cyberpanel version like below

![Image](https://github.com/user-attachments/assets/4bfde4ed-9a21-4ead-8f5f-8a8b1c103972)

**Comment by infinitywebtech007** [comment]
Now there is no option for privately hosted Git

**Comment by usmannasir** [comment]
Thanks for flagging this. The full self-hosted Git deployment screen has now been restored in the v3 site workspace under **Websites → List Websites → Manage → Advanced → Git Deployment**.

The custom-port handling is also fixed. Hosts entered as `git.example.com:2222` are now converted to the proper SSH URL form, while normal port 22 hosts keep the existing format. The fix is included in v3.0.1 and stable, and the release test suite passes all 160 tests.

Please rerun the public CyberPanel upgrader, then enter the Git host as `hostname:port`, the GitLab group/user as the username, and the repository name in the repository field. If it still fails, please paste the new deployment error (with tokens/keys removed) and we will trace that path as well.
